### PR TITLE
CDAP-15826 File-merge plugin validation - package rename

### PR DESCRIPTION
**Pull request by mlozbin-cybervisiontech**
JIRA: https://issues.cask.co/browse/CDAP-15826
In scope of this PR:
- renamed packages to io.cdap.plugin.filemerge.action
- added gitignore
- updated cdap version to 6.1.0-SNAPSHOT

### CODE REVIEW

**Comment by mlozbin-cybervisiontech** [comment]
@albertshau @CuriousVini could you, please, take a look.

File: pom.xml [hunk 1]
```diff
@@ -261,7 +261,7 @@
           <extensions>true</extensions>
           <configuration>
             <instructions>
-              <_exportcontents>co.cask.*;org.apache.spark.streaming.kafka.*;
+              <_exportcontents>io.cdap.plugin.filemerge.*;org.apache.spark.streaming.kafka.*;
                 kafka.serializer.*;kafka.common;org.apache.kafka.*;com.mapr.streams.*;
```
Comment (albertshau) [inline pom.xml]: these other exports look like they were copied over from another repo, please remove them. It should only need io.cdap.plugin.filemerge.*

**Review by albertshau — COMMENTED** [review]
one comment about removing some existing exports that should not be there. fix and squash.

File: pom.xml (hunk 1 above)
Comment (mlozbin-cybervisiontech) [reply pom.xml]: done

**Comment by mlozbin-cybervisiontech** [comment]
@albertshau I addressed your comment and squashed commits, could you, please, merge.

**Review by albertshau — APPROVED** [review]

**Pull request merged**